tools/oecert/host/host.cpp

```diff
@@ -546,10 +630,31 @@ static oe_result_t _process_params(oe_enclave_t* enclave)
             _params.verify,
             _params.verbose));
     }
-    else if (_params.generate_evidence)
+    else if (_params.generate_sgx_ecdsa)
```

Comment:
Maybe also add `SGX_LOCAL_ATTESTATION` format in this PR? And for local attestation, there are no endorsements related input/output.

Reply:
Local attestation needs input (target info) from verifier. Not a good format to be generated.